.github/workflows/build_and_test.yml

```diff
@@ -391,8 +391,9 @@ jobs:
       LC_ALL: C.UTF-8
       LANG: C.UTF-8
       PYSPARK_DRIVER_PYTHON: python3.9
+      PYSPARK_PYTHON: python3.9
```

Comment:
This aims to prevent the following error currently happening on this PR.
```
RuntimeError: Python in worker has different version 3.8 than that in driver 3.9, PySpark cannot run with different minor versions. Please check environment variables PYSPARK_PYTHON and PYSPARK_DRIVER_PYTHON are correctly set.
```

Reply:
👍